src/Simulation/Simulators.Tests/Circuits/Bug3758.qs

```diff
@@ -22,43 +20,31 @@ namespace Bug3758
     
     /// # Summary
     /// An example operation used for testing input transformations.
-    operation TransformationReferenceForward(register : Qubit[]) : Unit {
-        body (...) {
-            X(register[0]);
-            H(register[1]);
-            X(register[2]);
-        }
-        adjoint auto;
-        controlled auto;
-        controlled adjoint auto;
+    operation TransformationReferenceForward(register : Qubit[]) : Unit is Adj + Ctl {
+        X(register[0]);
+        H(register[1]);
+        X(register[2]);
     }
 
     /// # Summary
     /// An example operation used for testing input transformations.
-    operation TransformationReferenceReverse(register : Qubit[]) : Unit {
-        body (...) {
-            X(register[2]);
-            H(register[1]);
-            X(register[0]);
-        }
-        adjoint auto;
-        controlled auto;
-        controlled adjoint auto;
+    operation TransformationReferenceReverse(register : Qubit[]) : Unit is Adj + Ctl {
+        X(register[2]);
+        H(register[1]);
+        X(register[0]);
     }
 }
 
 
 
-namespace Microsoft.Quantum.Simulation.Simulators.Tests.Circuits
-{
+namespace Microsoft.Quantum.Simulation.Simulators.Tests.Circuits {
     open Bug3758;
-    open Microsoft.Quantum.Extensions.Testing;
+    open Microsoft.Quantum.Diagnostics;
 
     operation TransformedOperationTest() : Unit {
-        AssertOperationsEqualReferenced(
+        AssertOperationsEqualReferenced(3,
```

Comment:
The order of the arguments changed here, but I didn't see this callable's signature being changed in this PR. The  doc page for this operation [here](https://docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.diagnostics.assertoperationsequalreferenced) suggests that the number of qubits should be the first argument.

Reply:
I read this backwards. I see now that the new ordering is in line with the documentation. My bad!

Reply:
No worries, happy for the double-check on that!